deploy/docker/Dockerfile

```diff
@@ -26,7 +26,9 @@ COPY . /emqx
 ARG PKG_VSN
 ARG EMQX_NAME=emqx
 
-RUN cd /emqx && make $EMQX_NAME
+RUN cd /emqx \
+    && rm -rf _build/$EMQX_NAME/lib \
```

Comment:
delete the entire profile ?

Reply:
Other profiles will not be compiled or copied to the distribution, so I don't think it's necessary to delete